service/awsconfig/v5/resource/ebsvolume/mock_test.go

```diff
@@ -0,0 +1,46 @@
+package ebsvolume
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/aws-operator/service/awsconfig/v5/key"
```

Comment:
Lets please separate the import for the same project from third parties.

Reply:
Ack, missed that. Done.